lightning/src/ln/peer_handler.rs

```diff
@@ -569,16 +581,20 @@ impl<Descriptor: SocketDescriptor, RM: Deref, L: Deref> PeerManager<Descriptor,
 	/// generate error messages). Note that some other lightning implementations time-out connections
 	/// after some time if no channel is built with the peer.
 	///
+	/// `current_time` is used as an always-increasing counter that survives across restarts and is
```

Comment:
I don't see how it survives across restarts, that seems to imply persistence to me?

Reply:
Yeah, s'pose that works for me